tools/ci/actions/runner.sh

```diff
@@ -53,7 +53,8 @@ Build () {
   fi
   failed=0
   if grep -Fq ' warning: undefined variable ' build.log; then
-    echo Undefined Makefile variables detected
+    echo Undefined Makefile variables detected:
+    grep -F ' warning: undefined variable ' build.log
```

Comment:
would it make sense to `sort --unique` here, to avoid redundant warnings?

Reply:
I wasn't sure - when debugging these in the past, the number of them can in itself be useful, as well as the order as it can hint immediately that it's a recursive invocation problem, for example.